### ISSUE TITLE
Hiding milliseconds

### ISSUE THREAD

**Issue by lamplightdev** [fooloomanzoo/datetime-picker#9]
Hi - this is a wonderful component, thanks very much!

Is there a way to hide the milliseconds picker? Often when using this to pick a time, the milliseconds are not relevant.

Cheers.

**Comment by fooloomanzoo** [comment]
@lamplightdev Yes you are right. I was thinking about a `format-attribute` that let's you specify the format of time and date, that could be by default 

> `hh:mm:ss`

A localisation of culture areas that use `am/pm`-representations would be also good. In the next version I will try to implement that. Regards!

**Comment by lamplightdev** [comment]
Great work, thanks!

**Comment by johanbyren** [comment]
Is it possible to hide the seconds? I only need to select date and HH-mm.

**Comment by fooloomanzoo** [comment]
It's in the examples for <time-picker>. Have you tried to set clamp="seconds"

**Comment by johanbyren** [comment]
It worked. Thank you so much. Really like your date/time picker. 

Best regards
Johan
